libraries/standard/mqtt/utest/mqtt_config.h

```diff
@@ -20,7 +20,7 @@
 #endif
 
 #ifndef LIBRARY_LOG_LEVEL
-    #define LIBRARY_LOG_LEVEL    LOG_NONE
+    #define LIBRARY_LOG_LEVEL    LOG_ERROR
```

Comment:
Does this mean unit test results will be intertwined with error logs, or will they still be separate?. What's the advantage of this?

Reply:
It is a change based on @cobusve's code expressivity feedback about always having the error level as the default logging verbosity, and instead override with CLI/Build configuration instead of the other way round.